news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,143 @@
+
+While previous sentiments aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/en/Ranking_Criteria/Difficulty_naming#osu!taiko), this rework has deviated away from that goal, aiming to reflect true difficulty, regardless of naming. This also means that star rating values for maps with near-FCs are no longer near 9.5 stars, but rather can expand well into the 10 star range (medal hunters rejoice!).
+
+## Changes to performance points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values (around 40% lower). In addition, the survey indicated that the way performance points are calculated needed to be re-evaluated.
+
+Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance not only between difficulty-increasing mods, but their reductive counterparts.
+
+Served alongside the star rating rework, these changes aim to accurately reflect the new sentiments of the star rating system.
+
+### Globally applied changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which consists of both difficulty and accuracy performance points. This allows for simple balancing changes, and works alongside specific multipliers found within their respective separate difficulty values.
+
+- The HD multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty pp, rather than globally.
+- An EZ multiplier of **0.975x** has been added, to further nerf a problematic mod, especially when paired with DT.
+- The NF multiplier has been removed — as osu!taiko's HP system is different, it is not required nor needed, which also helps prevent sandbaggers in lower ranks.
+
+As well as performance point multiplier changes, the community survey highlighted issues with short maps being either underweighted at a full combo, or overweighted on plays with several misses.
+
+To combat this, misses will now significantly affect difficulty pp on maps with a max combo below 1000, scaling the miss count by a multiplier of `1000 / total successful hits`, and as such can significantly impact short maps with a higher miss count.
+
+![Effective miss count graph](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/effective-miss-count.jpg)
+
+### Changes to difficulty performance points values
+
+The following changes should result in impressive scores on harder beatmaps rewarding a significant increase of performance point values, with top player [syaron105](https://osu.ppy.sh/users/8741695) gaining over 6400 total pp!
+
+Please note that these changes only affect the difficulty part of performance, which refers to the star rating value under a mathematically scaled formula.
```

Comment:
what is "mathematically scaled formula" supposed to mean

Reply:
Removed this bit in cdede1d465f2efaa51b73a09709b30534fe2d6a6